src/libratbag.c

```diff
@@ -684,6 +684,7 @@ ratbag_create_resolution(struct ratbag_profile *profile, int index)
 	res->refcount = 0;
 	res->profile = profile;
 	res->index = index;
+	res->is_disabled = false;
```

Comment:
Does this set the default value? If not, where can I do this? The default should be false cause else we have to rewrite every driver.